playgrounds/react/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "react-playground",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "babel-jest": "^26.2.2",
```

Comment:
As these dependencies are not needed in production they should be added in the `devDependencies`. As such they will not be downloaded in production mode. I realise that this is mostly will not happen here but it is best practice and if someone copies your playground he could think that because they are in `dependencies` they are needed in production mode.

Reply:
Also, they are not needed because they are no tests

Reply:
Ok, I just followed the getting started of React to create a basic react application: https://create-react-app.dev/docs/getting-started/

Reply:
Allright ! As you wish it's a detail :)

Reply:
I removed them!